src/Organizer/EditOrganizerRestController.php

```diff
@@ -113,13 +114,29 @@ public function updateUrl($organizerId, Request $request)
     }
 
     /**
-     * @param $organizerId
+     * @deprecated Use updateName with language parameter instead.
```

Comment:
Handig voor als we later op `@deprecated` annotation willen zoeken voor opkuiswerk 👍